client/app/states/orders/details/details.state.js

```diff
@@ -31,9 +32,24 @@ function resolveOrder ($stateParams, CollectionsApi) {
 }
 
 /** @ngInject */
-function StateController (order, $state) {
+function resolveServiceTemplate ($stateParams, CollectionsApi) {
+  return CollectionsApi.get('service_orders', $stateParams.serviceOrderId, {
+    expand: ['resources', 'service_requests']
+  }).then((ServiceOrder) => {
+    const serviceTemplateId = ServiceOrder.service_requests[0].source_id;
+    return CollectionsApi.get('service_templates', serviceTemplateId, {
+      expand: ['resources'],
+      attributes: ['picture', 'resource_actions', 'picture.image_href'],
+    })
+  })
+}
+
+/** @ngInject */
+function StateController (order, serviceTemplate, $state) {
   const vm = this
   vm.order = order
+  vm.serviceTemplate = serviceTemplate
+  console.log(vm);
```

Comment:
This console log should probably be removed, right?

Reply:
Oh yeah, will remove and repush now